omod-common/src/main/java/org/openmrs/module/webservices/rest/web/ConversionUtil.java

```diff
@@ -58,6 +58,10 @@
 	
 	public static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
 	
+	private static final String DATE_TIME_PATTERN = "yyyy-MM-dd HH:mm";
+	
+	private static final String DATE_PATTERN = "yyyy-MM-dd";
```

Comment:
A couple of things here @mozzy11;
1) This date format seems to be already supported right?
2) If 1) is correct, why would we only introduce one more date format?

Reply:
oh sure , that is is aleady suppoerted. 
We introduce the other because ,the obs class already supports it , and there are test units that use it. 
Not supporting it would lead to several test Failures.
 Thats why innitialy i was a using a try and catch ,for the obs resource to delegate to the Rest conversion. so for the obs resource to completely delegate to the conversion Util without a try and catch , it must also be supporting that date format too.